Tangem/Modules/ManageTokensNetworkSelector/ManageTokensNetworkSelectorView.swift

```diff
@@ -12,10 +12,112 @@ struct ManageTokensNetworkSelectorView: View {
     @ObservedObject var viewModel: ManageTokensNetworkSelectorViewModel
 
     var body: some View {
-        VStack(spacing: 0) {
-            ForEach(viewModel.selectorItems) {
-                ManageTokensNetworkSelectorItemView(viewModel: $0)
+        ScrollView {
+            VStack(spacing: 0) {
+                Spacer(minLength: 10)
+
+                walletSelectorContent
+
+                if !viewModel.nativeSelectorItems.isEmpty {
+                    Spacer(minLength: 24)
+
+                    nativeNetworksContent
+                }
+
+                if !viewModel.nonNativeSelectorItems.isEmpty {
+                    Spacer(minLength: 24)
+
+                    noneNativeNetworksContent
+                }
+            }
+            .padding(.horizontal, 16)
+        }
+        .alert(item: $viewModel.alert, content: { $0.alert })
+        .navigationBarTitle(Text(Localization.manageTokensNetworkSelectorTitle), displayMode: .inline)
+        .background(Colors.Background.tertiary.edgesIgnoringSafeArea(.all))
+        .onAppear(perform: viewModel.onAppear)
+        .onDisappear(perform: viewModel.onDisappear)
+    }
+
+    private var walletSelectorContent: some View {
+        Group {
+            HStack(spacing: 16) {
+                Text(Localization.manageTokensNetworkSelectorWallet)
+                    .lineLimit(1)
+                    .style(Fonts.Regular.subheadline, color: Colors.Text.primary1)
+
+                Spacer(minLength: 0)
+
+                Text(viewModel.currentWalletName)
+                    .lineLimit(1)
+                    .style(Fonts.Regular.subheadline, color: Colors.Text.tertiary)
+
+                Assets.chevron.image
+                    .frame(width: 24, height: 24)
```

Comment:
resizable нужен для норм ресайза
```suggestion
                Assets.chevron.image
                .resizable()
                    .frame(width: 24, height: 24)
```